index.html

```diff
@@ -90,7 +90,7 @@ <h1><span class="auto-type"></span></h1>
         </div>
 
     </section>
-
+    <p>Contact me via Email, Message or connect me with social media accounts. Also send me any inquiries by filling in the form given just below</p>

```

Comment:
Remove the sentence from the index.html bcs index.html means the pre-loader of this website @theotsuaneng